CONTRIBUTING.md

```diff
@@ -41,7 +41,8 @@ and to meet some of our community members.
 3.  If you are comfortable with Git,
     and would like to add or change material,
     you can submit a pull request (PR).
-    Instructions for doing this are [included below](#using-github).
+    For information about using pull requests see 
+    [github documentation on using pull requests](https://help.github.com/articles/using-pull-requests/).
```

Comment:
We want to keep CONTRIBUTING.md the same across all repos, so please PR changes to it to the `styles` repo.